backend/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "esModuleInterop": true,
     "moduleResolution": "node",
     "sourceMap": true,
-    "rootDir": "./src",
+    // "rootDir": "./src",
```

Comment:
What does this do?

If intentional, just delete the whole thing rather comment it out.

Reply:
It was to remove error raised in visualStudioCode, I will remove it.